package.json

```diff
@@ -7,20 +7,22 @@
     "mysql": "^2.11.1"
   },
   "devDependencies": {
-    "babel": "^6.1.18",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.24.1",
     "babel-preset-es2015": "^6.1.18",
-    "istanbul": "^0.4.0",
     "chai": "^3.4.1",
     "coveralls": "^2.11.4",
     "esdoc": "^0.4.3",
+    "istanbul": "^0.4.0",
     "mocha": "^2.3.3",
     "mocha-lcov-reporter": "^1.0.0",
     "squel": "^5.6.0"
   },
   "scripts": {
+    "build": "babel src --out-dir lib",
```

Comment:
if you want to use the local babel-cli, change the command to `./node_modules/.bin/babel src --out-dir lib`